### ISSUE TITLE
autofitpanels does not respect header any more

### ISSUE THREAD

**Issue by kschlesselmann** [grafana/grafana#15650]
After upgrading to grafana 6.0.0 our dashboards contain scrollbars.

It seems that the header bar shown with `kiosk=tv` isn't used in the computations that are done during `autofitpanels` any more.

**Comment by jschill** [comment]
Hi! What do you mean by "contain scrollbars"? Do you get scrollbars in autofitmode? We've changed the component used for scrolling, which might have caused some side effect, but when I have Grafana 5.5 and 6.0 side by side I don't see any difference except the scrollbar design.

- What browser+version & os are you using?
- What version did you upgrade from?
- What is your journey to your dashboard? Do you click? Use a bookmark?
- Urls and screenshots are super helpful if possible. 

One thing to note about the scrollbars if you're on MacOS is there's a setting in `System Preferences > General` called "Show scroll bars" that might make it look different on different machines depending on your setting.

**Comment by covrig** [comment]
Same here (for kiosk=tv). The height is larger than the screen.
I just updated the Docker container from 6.0.1 beta to 6.0.0. Downgrading solves the problem. 
I see this on Windows (Chrome and IE) and Linux, no matter the journey to the dashboard.

**Comment by kschlesselmann** [comment]
@jschill I'm sorry I cannot provide any screenshots of our dashboards.
 All team members here experience the same behaviour though. So there's at least Win 10, Debian, Ubuntu, Manjaro and Arch involved with Fireforx, Chromium and Chrome.

Our installation updates nightly to the latest docker image.

Our journey to the dashbord is always a bookmark (we use scripted dashboards here)

The url looks like `https://.../dashboard/script/dashboard-1.js?environment=develop&orgId=1&refresh=30s&kiosk=tv&autofitpanels`

Basically I'd say we experience the same behaviour as @covrig 

Hope that's detail enough.

**Comment by jschill** [comment]
Thanks for reporting and input, @kschlesselmann && @covrig
